examples/pcb-aoi/incremental_learning_bench/testalgorithms/fpn/fpn_algorithm.yaml

```diff
@@ -0,0 +1,56 @@
+algorithm:
+  paradigm_kind: "incrementallearning"
+  incremental_learning_data_setting:
+    # ratio of training dataset; float type;
+    train_ratio: 0.8
+    # the method of splitting dataset; string type; optional;
+    # currently the options of value are as follows:
+    #   1> "default": the dataset is evenly divided based train_ratio;
+    splitting_method: "default"
+  # the url address of initial model for model pre-training; string url;
+  initial_model_url: "/ianvs/initial_model/model.zip"
+
+  # algorithm module configuration in the paradigm; list type;
+  modules:
+    # kind of algorithm module; string type;
+    # currently the options of value are as follows:
+    #   1> "basemodel": contains important interfaces such as train、 eval、 predict and more; required module;
+    - kind: "basemodel"
+      # name of python module; string type;
+      # example: basemodel.py has BaseModel module that the alias is "estimator" for this benchmarking;
+      name: "estimator"
+      # the url address of python module; string type;
+      url: "./examples/pcb-aoi/incremental_learning_bench/testalgorithms/fpn/basemodel.py"
+
+      # hyperparameters configuration for the python module; list type;
+      hyperparameters:
+        # name of the hyperparameter; string type;
+        - momentum:
+            # values of the hyperparameter; list type;
+            # types of the value are string/int/float/boolean/list/dictionary
+            values:
+              - 0.95
+              - 0.5
+          # hyperparameters configuration files; dictionary type;
+        - other_hyperparameters:
+            # the url addresses of hyperparameters configuration files; list type;
+            # type of the value is string;
+            values:
+              - "./examples/pcb-aoi/incremental_learning_bench/testalgorithms/fpn/fpn_hyperparameter.yaml"
+      #  2> "hard_example_mining": check hard example when predict ; optional module;
+    - kind: "hard_example_mining"
```

Comment:
It is suggested to explain why there are two modules, and also it is needed to provide a unified module definition. For example, the two module can be corresponding to the module in Sedna Incremental Learning. At least, it needs to be added about the description of how they now defined